app/views/episode_player/_form.html.erb

```diff
@@ -1,64 +1,123 @@
-<%= turbo_frame_tag "embed-player", data: {controller: "morphdom"} do %>
+<div class="row my-4 mx-2" data-controller="embed-preview"
```

Comment:
Bit aesthetic ... but I might break out some of the components into sub-partials?  `_form_player.html.erb` `_form_copy.html.erb` `_form_sidebar.html.erb` etc (naming is hard).

Just because this file is getting a bit lengthy + indented.

Reply:
Broke the elements shared by episode and podcast player forms into sub-partials. Podcast player form update to use episode player forms sub-partials.

Reply:
+1 nice!